var/spack/repos/builtin/packages/py-numpy/package.py

```diff
@@ -113,12 +113,6 @@ class PyNumpy(PythonPackage):
     # GCC 4.8 is the minimum version that works
     conflicts('%gcc@:4.7', msg='GCC 4.8+ required')
 
-    def setup_dependent_package(self, module, dependent_spec):
-        if 'python' in self.spec:
-            python_version = self.spec['python'].version.up_to(2)
-        else:
-            python_version = dependent_spec['python'].version.up_to(2)
-
```

Comment:
Just curious without knowing the impact : Is this also in upstream? (thinking if this being used in any dependency packages)

Reply:
Um, this code does not do anything at all. It's not in upstream, and all it does is to grab the python version. Probably a git merge thing I did not catch before.

Reply:
Checked, this was only in our code, but as I mention above, is code that does not do anything.